adhocteam/Head-Start-TTADP
# [TTAHUB-165] include roles w/author & collaborators on CSV

## Description of change
Used the calculated "fullName" field on the User model to populate the CSV exports. This field includes abbreviated roles, which are now included on the Creator & Collaborator fields on the CSV.

## How to test
Export a CSV, preferably one with collaborators. The Creator and Collaborators should be formatted as below.
- In CSV download, the Creator field contains [First Name] [Last Name], [role 1], [role...] 
- In CSV download, the Collaborators field contains [First Name][Last Name], [role 1], [role...] 

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-165

## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
